mmaction/datasets/pipelines/augmentations.py

```diff
@@ -1063,13 +1530,15 @@ class ColorJitter:
         eig_vec (np.ndarray | None): Eigenvectors of [3 x 3] size for RGB
             channel jitter. If set to None, it will use the default
             eigenvectors. Default: None.
+        apply_pca_noise (bool): Whether to apply pca noise. Default: True.
```

Comment:
Will this change affect the performance of all other checkpoints trained with no pca noise?